src/ext/oblivc/obliv_bits.h

```diff
@@ -2,6 +2,13 @@
 #define OBLIV_BITS_H
 #define __oblivious_c
 
+/* 
+   Note to maintainers: this header should *never* be included in any other
+   header; it will automatically be included in obliv-c source by the compiler,
+   and it contains defines that should never appear in plain C source (whereas
+   including most other obliv-c headers in regular C source is OK)
```

Comment:
Let's remove the part about most other headers. Yes it might be true today, but I don't want to encourage that.